resources/apt/apt.dsc.resource.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+export exist=true
+export NONINTERACTIVE=1
+
+# $packageName and $_exist are sent as env vars by dsc converting the JSON input to name/value pairs
+
+check_args() {
+    if [[ -z $packageName ]]; then
+        echo "packageName not set"
+        exit 1
+    fi
+}
+
+get_apt() {
+    pkgname=$1
+    InstalledSection=0
+    apt list --installed $pkgname 2>&1 | while read line; do
```

Comment:
Nit: Should this script use `apt` or `apt-get` - my understanding is that the latter is preferred for shell scripts and the former is more for interactive use.

Reply:
`apt-get list` is not valid on my system